.github/workflows/push_reproschema_py.yml

```diff
@@ -0,0 +1,50 @@
+name: Create Pull Request in Another Repository
+on:
+  push:
+    paths: 'releases'
+
+jobs:
+  create-pull-request:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v3
+      - name: Set up Git and cloning repository
+        env:
+          TARGET_REPO: repronim/reproschema-py
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        run: |
+          git config --global user.name "djarecka"
+          git config --global user.email "djarecka@gmail.com"
+          git clone https://x-access-token:${{ secrets.PERSONAL_ACCESS_TOKEN }}@github.com/$TARGET_REPO.git
+      - name: Make changes to target repository
+        id: changes
+        run: |
+          LAST_VERSION=$(ls -lt releases/ | grep ^d | head -1 | awk '{print $9}')
```

Comment:
how about replacing 
`LAST_VERSION=$(ls -lt releases/ | grep ^d | head -1 | awk '{print $9}')`
with the following
```
TARGET_DIR="releases"
URL="https://api.github.com/repos/${{ github.repository }}/contents/$TARGET_DIR?ref=main"
response=$(curl -sL -H "Authorization: token ${{ secrets.PERSONAL_ACCESS_TOKEN }}"  "$URL")
latest_version=$(echo $response | jq -r '.[0].name' | sort -rV | head -1)
```